roles/kube-burner/files/metrics-aggregated.yaml

```diff
@@ -163,3 +163,10 @@ metrics:
 
   - query: sum(kube_node_status_condition{status="true"}) by (condition)
     metricName: nodeStatus
+
+  - query: sum(rate(container_fs_writes_total[5m])) by (device, container, node)
```

Comment:
This query returns a bunch of timeseries. Roughtly number of containers * number of devices * number of nodes
This is a capture from a 75 node cluster.

>![image](https://user-images.githubusercontent.com/4614641/96987719-aa4ace80-1523-11eb-9517-8c4f5a5fc9ca.png)

As the goal of this query is to measure the disk usage made by etcd we can use this query instead:
>`sum(rate(container_fs_writes_total{container="etcd", device!~".+dm.+"}[5m])) by (device, container, node) > 0`
![image](https://user-images.githubusercontent.com/4614641/96988183-4aa0f300-1524-11eb-8fe9-bb00b001f2e6.png)

Reply:
This isn't what we are looking for though...

We could limit the query to just the master, but just etcd isn't helpful. This is to look for other containers who are competing for the disk.

Reply:
```
(sum(rate(container_fs_writes_total{device!~".+dm.+"}[5m])) by (device, container, node) and on (node) kube_node_role{role="master"}) >  0